### PR TITLE
feat: Allow convex partition on clockwise polygons

### PR DESCRIPTION
**Pull request by samestep**
Followup to https://github.com/penrose/penrose/pull/893#issuecomment-1055616275, made possible by https://github.com/x6ud/poly-partition-js/pull/1. Thanks @x6ud!

### CODE REVIEW

File: packages/core/src/contrib/__tests__/Minkowski.test.ts [hunk 1]
```diff
@@ -125,39 +125,39 @@ describe("convexPartitions", () => {
...
   test("Figure 1 from [HM83]", () => {
     // https://link.springer.com/content/pdf/10.1007/3-540-12689-9_105.pdf
-    // point locations approximated by tracing in Inkscape, then y-coordinates
-    // are inverted because screen coordinates are upside down compared to math,
-    // and the library we use expects the polygon to be counterclockwise
+    // point locations approximated by tracing in Inkscape; also note, this
+    // polygon is actually clockwise because screen coordinates are upside down
+    // compared to math
     const p = [
-      [-316.39758, 117.40885],
```
Comment (jiriminarcik) [inline packages/core/src/contrib/__tests__/Minkowski.test.ts:132]: Maybe we can leave this as well and test both orientations.

**Review by jiriminarcik — APPROVED** [review]
Wow, that was quick! :) Thanks @samestep and @x6ud!

File: packages/core/src/contrib/__tests__/Minkowski.test.ts (hunk 1 above)
Comment (samestep) [reply packages/core/src/contrib/__tests__/Minkowski.test.ts:132]: Hmm we could, but the other test is counterclockwise so we kind of already are testing both. Up to you though!

Comment (jiriminarcik) [reply packages/core/src/contrib/__tests__/Minkowski.test.ts:132]: It is now, but both tests will be a bit more future proof I think

Comment (samestep) [reply packages/core/src/contrib/__tests__/Minkowski.test.ts:132]: Fair enough! Done.

**Pull request merged**